def test_fail():
    assert 1 == 2  # Cố tình viết sai để CI fail
